### PR TITLE
Update oglor_devoted_assistant.txt

### LINKED ISSUE #3082: Oglor, Devoted Assistant "from library or hand" ability fix
Oglor, Devoted Assistant doesn't have "from your hand" part of its ability.

Added/Made changes:

Name:Oglor, Devoted Assistant
ManaCost:1 U
Types:Legendary Creature Homunculus
PT:1/1
T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ DBDig | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your upkeep, look at the top two cards of your library, then put one of them into your graveyard.
SVar:DBDig:DB$ Dig | ValidTgts$ You | DigNum$ 2 | ChangeNum$ 1 | DestinationZone$ Graveyard | LibraryPosition2$ 0
T:Mode$ ChangesZone | ValidCard$ Creature.YouOwn | Origin$ Library,Hand | Destination$ Graveyard | TriggerZones$ Battlefield | Execute$ TrigEffect | TriggerDescription$ Whenever a creature card is put into your graveyard from your library or hand, it perpetually gains "When this card leaves your graveyard, create a 2/2 tapped black Zombie creature token."
SVar:TrigEffect:DB$ Effect | RememberObjects$ TriggeredCard | StaticAbilities$ PerpetualAbility | Duration$ Permanent | Name$ Oglor, Devoted Assistant's Perpetual Effect
SVar:PerpetualAbility:Mode$ Continuous | AddTrigger$ LeavesGrave | Affected$ Card.IsRemembered | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ The card perpetually gains "When this card leaves your graveyard, create a 2/2 tapped black Zombie creature token."
SVar:LeavesGrave:Mode$ ChangesZone | Origin$ Graveyard | Destination$ Any | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When this card leaves your graveyard, create a 2/2 tapped black Zombie creature token.
SVar:TrigToken:DB$ Token | TokenScript$ b_2_2_zombie | TokenTapped$ True
DeckHas:Ability$Graveyard|Token & Type$Zombie
DeckHints:Ability$Graveyard|Mill & Keyword$Disturb|Escape
Oracle:At the beginning of your upkeep, look at the top two cards of your library, then put one of them into your graveyard.\nWhenever a creature card is put into your graveyard from your library or hand, it perpetually gains "When this card leaves your graveyard, create a 2/2 tapped black Zombie creature token."

Tested using 
forge-android-1.6.57-SNAPSHOT-05.08-signed-aligned

Tested with a card combo using:
• Dryad Arbor in hand
• Dryad Arbor in graveyard
• Tortured Existence on battlefield
• Bone Miser on battlefield
• Altar of the Brood on battlefield
• A way to generate one {B}

### PR DESCRIPTION
**Pull request by Simisays**
-closes #3082

### CODE REVIEW

File: forge-gui/res/cardsfolder/o/oglor_devoted_assistant.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ Types:Legendary Creature Homunculus
 PT:1/1
 T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ DBDig | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your upkeep, look at the top two cards of your library, then put one of them into your graveyard.
 SVar:DBDig:DB$ Dig | ValidTgts$ You | DigNum$ 2 | ChangeNum$ 1 | DestinationZone$ Graveyard | LibraryPosition2$ 0
-T:Mode$ ChangesZone | ValidCard$ Creature.YouOwn | Origin$ Library | Destination$ Graveyard | TriggerZones$ Battlefield | Execute$ TrigEffect | TriggerDescription$ Whenever a creature card is put into your graveyard from your library, it perpetually gains "When this card leaves your graveyard, create a 2/2 tapped black Zombie creature token."
+T:Mode$ ChangesZone | ValidCard$ Creature.YouOwn | Origin$ Library,Hand | Destination$ Graveyard | TriggerZones$ Battlefield | Execute$ TrigEffect | TriggerDescription$ Whenever a creature card is put into your graveyard from your library, it perpetually gains "When this card leaves your graveyard, create a 2/2 tapped black Zombie creature token."
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/o/oglor_devoted_assistant.txt:7]: but isn't that wrong? the trigger says only "Libary -> Graveyard", nothing on the card says Hand?

Comment (Hanmac) [reply forge-gui/res/cardsfolder/o/oglor_devoted_assistant.txt:7]: Ups i was wrong, The Trigger Descriptions needs to be updated too

**Review by tool4ever — APPROVED** [review]

File: forge-gui/res/cardsfolder/o/oglor_devoted_assistant.txt [hunk 2]
```diff
@@ -4,11 +4,11 @@ Types:Legendary Creature Homunculus
...
+T:Mode$ ChangesZone | ValidCard$ Creature.YouOwn | Origin$ Library,Hand | Destination$ Graveyard | TriggerZones$ Battlefield | Execute$ TrigEffect | TriggerDescription$ Whenever a creature card is put into your graveyard from your library or hand, it perpetually gains "When this card leaves your graveyard, create a tapped 2/2 black Zombie creature token."
 SVar:TrigEffect:DB$ Effect | RememberObjects$ TriggeredCard | StaticAbilities$ PerpetualAbility | Duration$ Permanent | Name$ Oglor, Devoted Assistant's Perpetual Effect
 SVar:PerpetualAbility:Mode$ Continuous | AddTrigger$ LeavesGrave | Affected$ Card.IsRemembered | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ The card perpetually gains "When this card leaves your graveyard, create a 2/2 tapped black Zombie creature token."
 SVar:LeavesGrave:Mode$ ChangesZone | Origin$ Graveyard | Destination$ Any | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When this card leaves your graveyard, create a 2/2 tapped black Zombie creature token.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/o/oglor_devoted_assistant.txt:10]: need to be changed there too

**Review by Hanmac — APPROVED** [review]
👍

**Pull request merged**